BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load(
+    "@build_bazel_rules_apple//apple:macos.bzl",
+    "macos_application",
+)
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+load(
+    "@build_bazel_rules_apple//apple:versioning.bzl",
+    "apple_bundle_version",
+)
+
+swift_library(
+    name = "Sources",
+    srcs = glob(["Sources/**/*.swift"]),
+    deps = [
+        "@AEXML//:AEXML", 
+        "@Commandant//:Commandant", 
+        "@Commander//:Commander", 
+        "@JSONUtilities//:JSONUtilities", 
+        "@PathKit//:PathKit", 
+        "@Rainbow//:Rainbow", 
+        "@Result//:Result", 
+        "@ShellOut//:ShellOut", 
+        "@Spectre//:Spectre", 
+        "@Tulsi//src/TulsiGenerator:tulsi_generator_lib",
+        "@XcodeGen//:XcodeGenKit", 
+        "@XcodeGen//:ProjectSpec", 
+        "@xcproj//:xcproj", 
+        "@Yams//:Yams"
+    ],
+    structured_resources = glob([
+        "XCHammerAssets/**"
+    ])
+)
+
+apple_bundle_version(
+    name = "XCHammerVersion",
+    build_version = "1.0",
+)
+
+macos_application(
+    name = "xchammer",
```

Comment:
So will we be able to simply run `bazel build xchammer`?

Reply:
Yup, bazel run requires more work since .apps aren’t considered executable for some reason